xml/Microsoft.Extensions.FileProviders.Internal/PhysicalDirectoryContents.xml

```diff
@@ -171,9 +172,17 @@
       </ReturnValue>
       <Parameters />
       <Docs>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <summary>Returns an enumerator that iterates through a collection.</summary>
+        <returns>An <see cref="T:System.Collections.IEnumerator" /> object that can be used to iterate through the collection.</returns>
```

Comment:
Can you please find all the places where the summary mentions `<see cref="T:System.Collections.IEnumerator" />` and simplify it to just "enumerator"?:

```suggestion
        <returns>An enumerator object that can be used to iterate through the collection.</returns>
```